draft-packages/avatar/KaizenDraft/Avatar/AvatarGroup.module.scss

```diff
@@ -0,0 +1,109 @@
+@import "~@kaizen/design-tokens/sass/typography";
+@import "~@kaizen/design-tokens/sass/shadow";
+@import "~@kaizen/design-tokens/sass/color";
+
+$avatar-sm: 1.25rem;
+$avatar-md: 3rem;
+$avatar-lg: 4.5rem;
+
+.AvatarGroup {
+  display: inline-flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.AvatarGroupItem,
+.AvatarCounter {
+  .small & {
+    margin-left: calc(#{$avatar-sm} / -2);
+  }
+
+  .medium & {
+    margin-left: calc(#{$avatar-md} / -2);
+  }
+
+  .large & {
+    margin-left: calc(#{$avatar-lg} / -2);
+  }
+
+  .small & [dir="rtl"] & {
+    margin-left: calc(#{$avatar-sm} / -2);
```

Comment:
Are these doing anything, they look the same as the LTR equivalents above?

Reply:
Absolutely right, it's redundant. I think that was left over from trying to square away LTR styles